website/docs/r/hpc_cache.html.markdown

```diff
@@ -66,6 +66,8 @@ The following attributes are exported:
 
 * `id` - The `id` of the HPC Cache.
 
+* `mount_addresses` - A list of mount addresses of the HPC Cache.
```

Comment:
[best I can tell](https://docs.microsoft.com/en-us/azure/hpc-cache/hpc-cache-mount) these are IP Addresses? can we clarify what these are:

```suggestion
* `mount_addresses` - A list of IP Addresses where the HPC Cache can be mounted.
```